lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -100,6 +100,7 @@ def script
 set -e
 
 mkdir -p "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+mkdir -p "${PODS_ROOT}/resources-checksums"
```

Comment:
In fact it's checksum of source resources and i think in a classic usage of cocoapods those resources files change only in a pod update